azure-pipelines.yml

```diff
@@ -1,342 +1,382 @@
+        terraformSubscriptionID: '8b6ea922-0862-443e-af15-6056e1c9b9a4'
+      - env: 'demo'
+        dependsOn: 'sbox'
+        serviceConnection: 'DCD-CFTAPPS-DEMO'
+        backendAzureRmResourceGroupName: 'azure-control-demo-rg'
+        aksKeyVault: 'dcdcftappsdemokv'
+        terraformSubscriptionID: 'd025fece-ce99-4df2-b7a9-b649d3ff2060'
+      - env: 'ithc'
+        dependsOn: 'sbox'
+        serviceConnection: 'DCD-CFTAPPS-ITHC'
+        backendAzureRmResourceGroupName: 'azure-control-ithc-rg'
+        aksKeyVault: 'dcdcftappsithckv'
+        terraformSubscriptionID: '62864d44-5da9-4ae9-89e7-0cf33942fa09'
+      - env: 'ptlsbox'
+        dependsOn: 'sbox'
+        serviceConnection: 'DTS-CFTSBOX-INTSVC'
+        backendAzureRmResourceGroupName: 'azure-control-ptlsbox-rg'
+        aksKeyVault: 'dtscftptlsbox'
+        terraformSubscriptionID: '1497c3d7-ab6d-4bb7-8a10-b51d03189ee3'
+      - env: 'perftest'
+        dependsOn: 'sbox'
+        serviceConnection: 'DCD-CFTAPPS-TEST'
+        backendAzureRmResourceGroupName: 'azure-control-test-rg'
+        aksKeyVault: 'dcdcftappstestkv'
+        terraformSubscriptionID: '8a07fdcd-6abd-48b3-ad88-ff737a4b9e3c'
+      - env: 'aat'
+        dependsOn: 'sbox'
+        serviceConnection: 'DCD-CFTAPPS-STG'
+        backendAzureRmResourceGroupName: 'azure-control-stg-rg'
+        aksKeyVault: 'dcdcftappsstgkv'
+        terraformSubscriptionID: '96c274ce-846d-4e48-89a7-d528432298a7'
+      - env: 'ptl'
+        dependsOn: 'ptlsbox'
+        serviceConnection: 'DTS-CFTPTL-INTSVC'
+        backendAzureRmResourceGroupName: 'azure-control-ptl-rg'
+        aksKeyVault: 'dtscftptl'
+        terraformSubscriptionID: '1baf5470-1c3e-40d3-a6f7-74bfbce4b348'
+      - env: 'prod'
+        dependsOn: 'aat'
+        serviceConnection: 'DCD-CFTAPPS-PROD'
+        backendAzureRmResourceGroupName: 'azure-control-prod-rg'
+        aksKeyVault: 'dcdcftappsprodkv'
+        terraformSubscriptionID: '8cbc6f36-7c56-4963-9d36-739db5d00b27'
 
 variables:
   - name: project
     value: cft
   - name: timeoutInMinutes
-    value: 60
+    value: "60"
   - name: agentPool
-    value: ubuntu-20.04 
-  - name: serviceConnection
-    ${{ if eq( parameters['env'], 'perftest') }}: 
-      value: "DCD-CFTAPPS-TEST"
-    ${{ if eq( parameters['env'], 'preview') }}: 
-      value: "DCD-CFTAPPS-DEV"
-    ${{ if eq( parameters['env'], 'aat' ) }}: 
-      value: "DCD-CFTAPPS-STG"
-    ${{ if eq( parameters['env'], 'ptlsbox' ) }}: 
-      value: "DTS-CFTSBOX-INTSVC"
-    ${{ if eq( parameters['env'], 'ptl' ) }}: 
-      value: "DTS-CFTPTL-INTSVC"
-    ${{ if notIn( parameters['env'], 'aat', 'perftest', 'ptlsbox', 'preview', 'ptl') }}: 
-      value: "DCD-CFTAPPS-${{ parameters.env }}"
+    value: ubuntu-20.04
   - name: tfInitSub
     value: "04d27a32-7a07-48b3-95b8-3c8691e1a263"
   - name: isMain
     value: $[eq(variables['Build.SourceBranch'], 'refs/heads/main')]
   - name: product
-    value: 'cft-platform'
+    value: cft-platform
```

Comment:
are all the cosmetic changes needed? Example:
'' to ""
"" to ''
'' to nothing

Reply:
yes as before it was using double quote where it was needed and single quote too